inspirehep/modules/records/jsonschemas/records/hep.json

```diff
@@ -684,7 +684,44 @@
         },
         "references": {
             "items": {
-                "$ref": "elements/reference.json"
+                "properties": {
+                    "curated_relation": {
+                        "type": "boolean"
+                    },
+                    "raw_refs": {
+                        "items": {
+                            "properties": {
+                                "position": {
+                                    "description": "The position of the reference as it appears on the document",
+                                    "type": "string"
+                                },
+                                "schema": {
+                                    "description": "E.g. text, or Elsevier XML, or refexract",
+                                    "type": "string"
+                                },
+                                "source": {
+                                    "type": "string"
+                                },
+                                "value": {
+                                    "type": "string"
+                                }
+                            },
+                            "required": [
+                                "value",
+                                "schema"
+                            ],
+                            "type": "object"
+                        },
+                        "type": "array"
+                    },
+                    "record": {
+                        "$ref": "elements/json_reference.json"
```

Comment:
This is the mini-record as designed by @mihaibivol . I am not yet fully convinced it's worth putting it in the schema because I believe it shouldn't be editable by cataloguers, rather generated automatically from raw_refs. Cataloguers should focus only on matching existing references.